modules/sagemaker/sagemaker-model-cicd/bin/sagemaker-model-cicd.ts

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env node
+import * as cdk from 'aws-cdk-lib';
+import { AwsSolutionsChecks } from 'cdk-nag';
+import 'source-map-support/register';
+import { MLOpsCodePipelineStack } from '../lib';
+import {
+  getModuleInfo,
+  getModuleParameters,
+  loadSeedFarmerEnvVars,
+} from './seedfarmer-parameters';
+
+// load seedfarmer environment variables from cache for codepipeline deployments
+loadSeedFarmerEnvVars();
+
+const account = process.env.CDK_DEFAULT_ACCOUNT;
+const region = process.env.CDK_DEFAULT_REGION;
+const { sfProjectName, sfDeploymentName, sfModuleName } = getModuleInfo();
+const moduleParameters = getModuleParameters();
+
+const app = new cdk.App();
+const stackName = moduleParameters.projectName;
+new MLOpsCodePipelineStack(app, stackName, {
+  env: {
+    account,
+    region,
+  },
+  tags: {
+    SeedFarmerProjectName: sfProjectName,
+    SeedFarmerDeploymentName: sfDeploymentName,
+    SeedFarmerModuleName: sfModuleName,
+  },
```

Comment:
This is redundant if you're adding them on the app level. When you add tags to the app, it automatically gets added to all the stacks.

Reply:
These are used to tag stacks that are created as part of code pipeline stages. The tags added at the app level doesn't seem to be propagated to those.